Enable conversion from URDF content (#158)
cyberbotics/urdf2webots: README.md

The script accepts the following arguments:
  - **-h, --help**: Show the help message and exit.

>   - **--input=INPUT**: Specifies the URDF file to convert or the URDF content string.

Reviewer: Instead of having a dual semantics for the `--input` parameter, it would be better to support only the file variant (not the the content string) and to read the content from `stdin` in case the `--input` command line is missing.

  - **--robot-name**: Specify the name of the robot and generate a Robot node string instead of a PROTO file (has to be unique).
  - **--normal**: If set, the normals are exported if present in the URDF definition.
  - **--box-collision**: If set, the bounding objects are approximated using boxes.
  - **--tool-slot=LinkName**: Specify the link that you want to add a tool slot to (exact link name from URDF, for PROTO conversion only).
  - **--translation="0 0 0"**: Set the translation field of the PROTO file or Webots Robot node string.
  - **--rotation="0 0 1 0"**: Set the rotation field of the PROTO file or Webots Robot node string.
  - **--init-pos=JointPositions**: Set the initial positions of your robot joints. Example: `--init-pos="[1.2, 0.5, -1.5]"` would set the first 3 joints of your robot to the specified values, and leave the rest with their default value.
  - **--link-to-def**: Creates a DEF with the link name for each solid to be able to access it using getFromProtoDef(defName) (for PROTO conversion only).
  - **--joint-to-def**: Creates a DEF with the joint name for each joint to be able to access it using getFromProtoDef(defName) (for PROTO conversion only).

> In case of the **--input** command line is missing, the conversion tool will read the content of `stdin`. You might simply copy-past the content of your URDF file in the terminal. Unfortunately the conversion tool will not be able to deal with relative paths present in your URDF file if you chose to not specify **--input**.

Reviewer: ```suggestion
In case the **--input** option is missing, the script will read the URDF content from `stdin`.
In that case, you can pipe the content of your URDF file into the script: `cat my_robot.urdf | urdf2proto.py`.
Relative paths present in your URDF file will be treated relatively to the current directory from which the script is called.
```

| Terminal   |      Python      |
|----------|-------------|
| --input |  input |

> | --output |  outFile |

Reviewer: changing from `inFile` to `input` is fine, it is more consistent. However we should also do the same for `outFile` and `output`.

| --link-to-def |  linkToDef |
| --joint-to-def |  jointToDef |

> In Python, you are able to convert your URDF file by using its path as input to the `convertUrdfFile()` function or directly by giving its content as input to the `convertUrdfContent()` function.

Reviewer: ```suggestion
In Python, you can convert a URDF file by passing its path as an argument to the `convertUrdfFile()` function or directly by passing its content as an argument to the `convertUrdfContent()` function.
```

> Once again, the conversion tool will not be able to deal with relative paths present in your URDF file if you work with the `convertUrdfContent()` function.
> 

Reviewer: ```suggestion
```
